components/automate-ui/src/app/modules/desktop/check-in-time-series/check-in-time-series.component.html

```diff
@@ -1,7 +1,13 @@
 <div class="content-container">
   <div class="header">
     <div class="heading">
-      <h2 class="heading-title">Check-in History</h2>
+      <div class="heading-title-container">
```

Comment:
I added a container around the title because I prefer to use flex for alignment purposes.  You will see this in all the html files